<!DOCTYPE html><html lang="en">
<head><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1">
<title>Gao Lab @ BIG, CAS</title><meta name="description" content="Page-specific description"><meta name="keywords" content="Page-specific keywords">
<link rel="shortcut icon" href="favicon.ico">
<link rel="shortcut icon" href="static/images/favicons/favicon.png" type="image/x-icon">
<link rel="apple-touch-icon" href="static/images/favicons/apple-touch-icon.png">
<link rel="apple-touch-icon-precomposed" href="static/images/favicons/apple-touch-icon-precomposed.png">
<link href="https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,300,400,600,700,800%7COpen+Sans+Condensed:300&amp;subset=latin,cyrillic" rel="stylesheet" type="text/css">
<link rel="stylesheet" href="static/css/screen.min.css"><link rel="stylesheet" href="static/css/custom.min.css"><script src="static/js/head.min.js">
</script>
</head>

<body>
<!-- Header-->
<header class="main-header">
<div class="first-nav">
<div class="container">
<nav class="first-nav__links">
<ul><li><a class="is-selected">Gao Lab @ BIG</a></li>
</ul>
</nav>
<a class="first-nav__navcontrol icon icon_menu js-navcontrol" href="#">Menu</a>
</div>
</div>

<div class="second-nav js-nav"><div class="container">
<a class="second-nav__logo">
<span>GY</span>
</a><nav class="menu">
<ul>
<li>
	<a class="js-link js-anchor is-selected" href="#home">Home</a>
</li>
<li>
	<a class="js-link js-anchor" href="#about">Research</a>
</li>
<li>
	<a class="js-link js-anchor" href="#team">Member</a>
</li>
<li>
	<a class="js-link js-anchor" href="#services">Software</a>
</li>
<li>
	<a class="js-link js-anchor" href="#projects">Gallery</a>
</li>
<li>
	<a class="js-link js-anchor" href="#testimonials">Publications</a>
</li>
<li>
	<a class="js-link js-anchor" href="#process">Openings</a>
</li>
<li>
	<a class="js-link js-anchor" href="#contacts">Contacts</a>
</li>
</ul>
</nav>
</div>
</div>
</header>

<!-- Home section-->
<section class="home-section" id="home">
<div class="home-section__bg">
</div>
<div class="home-section__overlay">
<div class="container">
<header class="home-section__header">
	<p class="home-section__header-label"><b>We Are</b></p>
	<h1>ATCG Lab</h1>
	<h3 style="color:white;"><b>(Advanced Technologies for Computational Genomics)</b></h3>
</header>
</div>
</div>

</section><!-- About section--><section class="about-section" id="about">
<div class="container">
<div class="about-section__card">
<header class="about-section__card-header">
	<h2>Welcome to Gao Lab!</h2>
</header>
<div class="about-section__card-info">
<p>Our lab is broadly interested in the development and application of novel computational methods combined with the latest sequencing technologies to solve a variety of biological problems, including but not limited to the following areas. </p>
<blockquote><p>1) Metagenomics and microbiome;</p></blockquote>
<blockquote><p>2) Bulk and single-cell transcriptomics;</p></blockquote>
<blockquote><p>3) Nucleic acid modification;</p></blockquote>
<blockquote><p>4) Long-read sequencing data processing and analysis. </p></blockquote>
<p>The recent research of Dr. Yuan Gao focused on development of computational methods for omics data, including several widely used tools in transcriptomics and metagenomics: </p>
<blockquote><p>1) a de novo method for sensitive and specific detection of circular RNAs, CIRI/CIRI2; </p></blockquote>
<blockquote><p>2) a method for detecting internal structure and alternative splicing events for circular RNAs, CIRI-AS; </p></blockquote>
<blockquote><p>3) a reference-free method for quantifying and comparing bacterial growth dynamics from multiple metagenomic samples, DEMIC; </p></blockquote>
<blockquote><p>4) a method for robust discovery and quantification of transcript isoform from error-prone long-read RNA-seq data, ESPRESSO. </p></blockquote>
<p>Dr. Gao has an extensive publication record in recent years, including corresponding or first author papers published in top journals such as <i><b>Nature Methods,</b></i> <i><b>Science Advances,</b></i> <i><b>Nature Communications,</b></i> <i><b>Advanced Science,</b></i> <i><b>Genome Biology</b></i> and <i><b>Trends in Genetics</b></i>, which have more than 3,000 citations according to Google Scholar. In 2021, Dr. Gao set up this laboratory in Beijing Institute of Genomics (BIG), Chinese Academy of Sciences (CAS). </p>
</div>
</div>
</div>

<!-- Team section--><section class="common-section" id="team"><div class="container">
<header class="common-section__header">
	<h2>Lab Members</h2>
</header>
<div class="team">
  <div class="team__member">
    <img src="static/images/team/YuanGao.jpg" alt="Yuan Gao">
      <div class="team__member-info">
        <h3 class="team__member-name">Yuan Gao</h3>
          <p class="team__member-role">Principal Investigator</p>
            <p class="team__member-descr">Dr. Yuan Gao is a professor at Beijing Institute of Genomics (BIG), Chinese Academy of Sciences (CAS). He received his Ph.D. degree at university of CAS, and completed his postdoctoral training at University of Pennsylvania (UPENN) and Childrenâ€™s Hospital of Philadelphia (CHOP).                   </p>
  </div></div><div class="team__member">
    <img src="static/images/team/LipuSong.jpg" alt="Lipu Song">
      <div class="team__member-info">
        <h3 class="team__member-name">Lipu Song</h3>
          <p class="team__member-role">Research Assistant Professor</p>
            <p class="team__member-descr">Dr. Lipu Song is the lab manager of Gao lab. He received his Ph.D. degree in bioinformatics at Institute of Genomics (BIG), CAS. His research interests include developing and applying computational methods to explore genome, transcriptome and other biology problems. </p>
              </div></div>
      <div class="team__member">
        <img src="static/images/team/ZihanZhou.jpg" alt="Zihan Zhou">
          <div class="team__member-info">
            <h3 class="team__member-name">Zihan Zhou</h3>
              <p class="team__member-role">PhD Student</p>
                <p class="team__member-descr">Zihan Zhou is a Ph.D. student in Gao lab majored in bioinformatics. She completed her master's degree in Genomics at BIG, CAS. She is interested in developing predictive AI algorithms, and she is now working on using deep learning model to understand RNA splicing pattern.                       </p></div></div>
      <div class="team__member"><img src="static/images/team/XinZheng.jpg" alt="Xin Zheng">
        <div class="team__member-info">
          <h3 class="team__member-name">Xin Zheng</h3>
            <p class="team__member-role">PhD Student</p>
              <p class="team__member-descr">Xin Zheng completed his bachelor's degree in Biotechnology from Harbin Medical University. Now he is majoring in genomics at BIG, CAS. He is interested in deep learning and its application in circRNA identification as well as other genomic fields.                                                  </p></div></div>
      <div class="team__member"><img src="static/images/team/WenaoHu.jpg" alt="Wen'ao Hu">
        <div class="team__member-info">
          <h3 class="team__member-name">Wen'ao Hu</h3>
            <p class="team__member-role">Master Student</p>
              <p class="team__member-descr">Wen'ao Hu is a master student at BIG, CAS. He completed his bachelor's degree from Huazhong Agricultural University. He is interested in development of high-precision method for strain identification and growth dynamics in metagenomics.                                  </p></div></div>
      <div class="team__member">
        <img src="static/images/team/WeizhiYuan.jpg" alt="Weizhi Yuan">
          <div class="team__member-info">
            <h3 class="team__member-name">Weizhi Yuan</h3>
              <p class="team__member-role">PhD Student</p>
                <p class="team__member-descr">Weizhi Yuan graduated from Southeast University and received his bachelor's degree in 2021. He is majoring in genomics at BIG, CAS from 2022. He is interested in applying machine learning algorithms for identification of nucleic acid modification.                                                              </p></div></div>
      <div class="team__member">
        <img src="static/images/team/BingqiWu.jpg" alt="Bingqi Wu">
          <div class="team__member-info">
            <h3 class="team__member-name">Bingqi Wu</h3>
              <p class="team__member-role">PhD Student</p>
                <p class="team__member-descr">Bingqi Wu finished her bachelor's degree at South China Agricultural University in 2022. She is majoring in Bioinformatics at BIG, CAS. Now, she is actively learning the relevant knowledge of programming and algorithms to lay a solid foundation for her future research on genomics and spatial transcriptomics.</p></div></div>
      <div class="team__member">
        <img src="static/images/team/ShixinChen.jpg" alt="Shixin Chen">
          <div class="team__member-info">
            <h3 class="team__member-name">Shixin Chen</h3>
              <p class="team__member-role">PhD Student</p>
                <p class="team__member-descr">Shixin Chen graduated from Wuhan University with a bachelor's degree. He majors in bioinformatics at BIG, CAS. His research mainly concentrates on the basic research and frontier issues of bioinformatics, such as method development and spatial transcriptomics.                                                 </p></div></div>
      <div class="team__member">
        <img src="static/images/team/MengxuePeng.jpg" alt="Mengxue Peng">
          <div class="team__member-info">
            <h3 class="team__member-name">Mengxue Peng</h3>
              <p class="team__member-role">Postdoc Researcher</p>
                <p class="team__member-descr">Dr. Mengxue Peng is now a postdoctoral fellow in the Gao lab. She received her Ph.D. degree from the Department of Food Science and Nutritional Engineering, China Agricultural University. She is interested in the interaction mechanisms between tumor microbiome and host.                                                </p></div></div>
      <div class="team__member">
        <img src="static/images/team/WeiPan.jpg" alt="Wei Pan">
          <div class="team__member-info">
            <h3 class="team__member-name">Wei Pan</h3>
              <p class="team__member-role">PhD Student</p>
                <p class="team__member-descr">Wei Pan is a Ph.D. student in the Gao lab, majoring in bioinformatics. He received his master's degree from Shandong University. He is interested in single cell spatial transcriptomics and is currently developing tools for full-length transcript isoforms.                                                 </p></div></div>
      <div class="team__member">
        <img src="static/images/team/JieZhang.jpg" alt="Jie Zhang">
          <div class="team__member-info">
            <h3 class="team__member-name">Jie Zhang</h3>
              <p class="team__member-role">Master Student</p>
                <p class="team__member-descr">Jie Zhang graduated from Ocean University of China and received her bachelor's degree in 2023. Since 2023, she has been studying bioinformatics at the Beijing Institute of Genomics, Chinese Academy of Sciences (BIG, CAS). Her research interest is in the field of metatranscriptomics.                                               </p></div></div>
      <div class="team__member">
        <img src="static/images/team/YukunZhang.jpg" alt="Yukun Zhang">
          <div class="team__member-info">
            <h3 class="team__member-name">Yukun Zhang</h3>
              <p class="team__member-role">Master Student</p>
                <p class="team__member-descr">Yukun Zhang completed his bachelor's degree at Northwest A&F University in 2023. He is majoring in bioinformatics at BIG, CAS, and is passionate about using deep learning algorithms to unravel the mysteries of microbiome data.                                                 </p></div></div>
       <div class="team__member">
        <img src="static/images/team/LiangTie.jpg" alt="Liang Tie">
          <div class="team__member-info">
            <h3 class="team__member-name">Liang Tie</h3>
              <p class="team__member-role">Master Student</p>
                <p class="team__member-descr">Liang Tie graduated from Southern Medical University in 2024 with a bachelor's degree. Now she is studying at the Sino-Danish College of the University of Chinese Academy of Sciences, majoring in Bioinformatics. She is interested in the alternative splicing of non-coding RNAs and their roles in diseases.  </p></div></div>
       <div class="team__member">
        <img src="static/images/team/ShuaiZhang.jpg" alt="Shuai Zhang">
          <div class="team__member-info">
            <h3 class="team__member-name">Shuai Zhang</h3>
              <p class="team__member-role">Master Student</p>
                <p class="team__member-descr">Shuai Zhang graduated from Northeast Forestry University in 2024. He is majoring in Bioinformatics at BIG, CAS, focusing on using deep learning algorithms to solve biological problems.</p></div></div>
</div></div></section>


</section><!-- Services section--><section class="common-section" id="services"><div class="container">
<header class="common-section__header">
	<h2>Software</h2>
</header>
<div class="grid-container">
	<ul class="services services_advisory">
		<li class="services__item icon icon_strategy">
			<h4>ESPRESSO</h4>
			<p>(<b>e</b>rror <b>s</b>tatistics <b>pr</b>omoted <b>e</b>valuator of <b>s</b>plice <b>s</b>ite <b>o</b>ptions)</p>
			<p>Transcript discovery and quantification from long-read RNA-seq data</p>
			<p><a href='https://github.com/Xinglab/espresso'>https://github.com/Xinglab/espresso</a></p>
		</li>
		<li class="services__item icon icon_strategy">
			<h4>DEMIC</h4>
			<p>(<b>d</b>ynamic <b>e</b>stimator of <b>mi</b>crobial <b>c</b>ommunities)</p>
			<p>Quantification and comparison of bacterial growth rates in multiple metagenomic samples</p>
			<p><a href='https://sourceforge.net/projects/demic/files/'>https://sourceforge.net/projects/demic/files/</a></p>
		</li>
		<li class="services__item icon icon_strategy">
			<h4>CRISPR-network</h4>
			<p>Reconstruction of phage-bacterium interaction based on CRISPR sequence structure</p>
			<p><a href='http://159.226.67.237:8080/lab_update/Newe/tools.jsp'>http://159.226.67.237:8080/lab_update/Newe/tools.jsp</a></p>
		</li>
		<li class="services__item icon icon_strategy">
			<h4>SPIRIT</h4>
			<p>(<b>sp</b>lint <b>i</b>mproved <b>r</b>epeat <b>i</b>dentifier for <b>t</b>ranscripts)</p>
			<p>Obtaining consensus sequences from long concatemeric cDNA reads with known splint sequences</p>
			<p><a href='https://github.com/Xinglab/SPIRIT/tree/main/SPIRIT'>https://github.com/Xinglab/SPIRIT/tree/main/SPIRIT</a></p>
		</li>
	</ul>
	<ul class="services services_pm">
		<li class="services__item icon icon_strategy">
			<h4>CIRI</h4>
			<p>(<b>ci</b>rcular <b>R</b>NA <b>i</b>dentifier)</p>
			<p>Identification of circular RNAs (circRNAs) from high-throughput RNA-seq data</p>
			<p><a href='http://sourceforge.net/projects/ciri/files/'>http://sourceforge.net/projects/ciri/files/</a></p>
		</li>
		<li class="services__item icon icon_strategy">
			<h4>CIRI-AS</h4>
			<p>Identification and quantification of internal structure and alternative splicing in circRNAs</p>
			<p><a href='https://sourceforge.net/projects/ciri/files/CIRI-AS'>https://sourceforge.net/projects/ciri/files/CIRI-AS</a></p>
		</li>
		<li class="services__item icon icon_strategy">
			<h4>CIRI2</h4>
			<p>Improved identification of circRNAs based on multiple seed matching</p>
			<p><a href='https://sourceforge.net/projects/ciri/files/CIRI2'>https://sourceforge.net/projects/ciri/files/CIRI2</a></p>
		</li>
		<li class="services__item icon icon_strategy">
			<h4>CIRI3</h4>
			<p>Detecting and quantifying circRNAs in terabyte-scale RNA-seq datasets</p>
			<p><a href='https://github.com/gyjames/CIRI3'>https://github.com/gyjames/CIRI3</a></p>
			<p><a href='https://ngdc.cncb.ac.cn/bit/ciri3'>https://ngdc.cncb.ac.cn/bit/ciri3</a></p>
		</li>
		<li class="services__item icon icon_strategy">
			<h4>CIRI-deep</h4>
			<p>Deep learning model for prediction of differentially spliced circRNAs from low-depth, polyA-selected, single-cell and spatial transcriptomics data</p>
			<p><a href='https://github.com/gyjames/CIRI-deep'>https://github.com/gyjames/CIRI-deep</a></p>
			<p><a href='https://ngdc.cncb.ac.cn/bit/ciriDeep'>https://ngdc.cncb.ac.cn/bit/ciriDeep</a></p>
		</ul>
</div></div></section>

<!-- Projects section--><section class="common-section" id="projects">
<header class="common-section__header">
	<h2>Gallery</h2>
</header>
<div class="projects-wrap"><div class="grid-container"><div class="projects">
	<figure class="projects__item"><img src="static/images/projects/PTR.jpg" alt="Bidirectional DNA replication of Bacterial genome and peak-to-trough ratio (PTR)"><figcaption><a class="icon icon_link" href="#testimonials" rel="nofollow">Bidirectional DNA replication of Bacterial genome and peak-to-trough ratio (PTR)</a></figcaption></figure>
	
	<figure class="projects__item"><img src="static/images/projects/CIRI_deep.jpg" alt="Features and outputs of CIRI-deep models"><figcaption><a class="icon icon_link" href="#testimonials" rel="nofollow">Features and outputs of CIRI-deep models</a></figcaption></figure>
	<figure class="projects__item"><img src="static/images/projects/CIRI_AS_pathways.jpg" alt="Enrichment analysis for fruit fly transcriptomics"><figcaption><a class="icon icon_link" href="#testimonials" rel="nofollow">Enrichment analysis for fruit fly transcriptomics</a></figcaption></figure>
	<figure class="projects__item"><img src="static/images/projects/ESPRESSO.jpg" alt="Icon of ESPRESSO for transcript isoform identification"><figcaption><a class="icon icon_link" href="#testimonials" rel="nofollow">Icon of ESPRESSO for transcript isoform identification</a></figcaption></figure>
	
	<figure class="projects__item"><img src="static/images/projects/DEMIC.jpg" alt="Iteration in DEMIC algorithm for quantifying PTR"><figcaption><a class="icon icon_link" href="#testimonials" rel="nofollow">Iteration in DEMIC algorithm for quantifying PTR</a></figcaption></figure>
	<figure class="projects__item"><img src="static/images/projects/CIRIdeep_IG.jpg" alt="Integrated gradient of cis and trans feature of CIRI-deep"><figcaption><a class="icon icon_link" href="#testimonials" rel="nofollow">Integrated gradient of cis and trans feature of CIRI-deep</a></figcaption></figure>


	<figure class="projects__item"><img src="static/images/projects/IR_quantify.jpg" alt="Intron retention quantification based on long and short read data"><figcaption><a class="icon icon_link" href="#testimonials" rel="nofollow">Intron retention quantification based on long and short read data</a></figcaption></figure>
	<figure class="projects__item"><img src="static/images/projects/SJ_inaccuracy.jpg" alt="Accuracy of perfectly and imperfectly aligned splice junctions output by map tool"><figcaption><a class="icon icon_link" href="#testimonials" rel="nofollow">Iteration in DEMIC algorithm for quantifying PTR</a></figcaption></figure>

</div></div></div></section>


<!-- Testimonials section--><section class="common-section" id="testimonials"><div class="container"><header class="common-section__header">
<h2>Publications</h2>
</header>
<div class="grid-container"><blockquote>
<ul class="testimonials">
	<li class="testimonials__item"><p><b>Zheng X^</b>, Zhang J^, <b>Song L^</b>, Li X, Zhao F*, <b>Gao Y*</b>. Detecting and quantifying circular RNAs in terabyte-scale RNA-seq datasets with CIRI3 [J]. <b>Nature Biotechnology</b>, 2025, DOI: 10.1038/s41587-025-02835-1. </p></li>
	<li class="testimonials__item"><p><b>Zhou Z^</b>, Zhang J^, <b>Zheng X</b>, Pan Z, Zhao F*, <b>Gao Y*</b>. CIRI-deep enables single-cell and spatial transcriptomic analysis of circular RNAs with deep learning [J]. <b>Advanced Science</b>, 2024, DOI: 10.1002/advs.202308115. </p></li>
	<li class="testimonials__item"><p><b>Zhang W^</b>, Li X^, Liu F, <b>Zhang J</b>, Tian J, <b>Gao Y*</b>. Fungen: clustering and correcting long-read metatranscriptomic data for exploring eukaryotic microorganisms [J]. <b>Science China Life Sciences</b>, 2025, DOI: 10.1007/s11427-024-2853-x.</p></li>
  <li class="testimonials__item"><p><b>Gao Y*^</b>, Wang F^, Wang R^, Kutschera E, Xu Y, Xie S, Wang Y, Kadash-Edmondson K, Lin L & Xing Y*. ESPRESSO: Robust discovery and quantification of transcript isoforms from error-prone long-read RNA-seq data. <b>Science Advances</b>, 2023, 9(3), eabq5072. </p></li>
	<li class="testimonials__item"><p>Xin R, Gao Y, <b>Gao Y</b>, Wang R, Kadash-Edmondson K & Xing Y. isoCirc catalogs full-length circular RNA isoforms in human transcriptomes [J]. <b>Nature Communications</b>, 2021, 12(1): 1-11.</p></li>
	<li class="testimonials__item"><p>Tanes C, Bittinger K, <b>Gao Y</b>, Friedman S E, Nessel L, RoyPaladhi U, Chau L, Panfen E, Fischbach M A, Braun J, Xavier R J, Clish C B, Li H, Bushman F D, Lewis J D & Wu G D. Role of dietary fiber in the recovery of the human gut microbiome and its metabolome [J]. <b>Cell Host & Microbe</b>, 2021, 29(3): 394-407. e5. </p></li>
	<li class="testimonials__item"><p><b>Gao Y</b>, Li H. Quantifying and comparing bacterial growth dynamics in multiple metagenomic samples [J]. <b>Nature Methods</b>, 2018, 15(12): 1041.</p></li>
	<li class="testimonials__item"><p><b>Gao Y</b>, Zhao F. Computational strategies for exploring circular RNAs [J]. <b>Trends in Genetics</b>, 2018, 34(5): 389-400. </p></li>
	<li class="testimonials__item"><p><b>Gao Y</b>, Zhang J, Zhao F. Circular RNA identification based on multiple seed matching [J]. <b>Briefings in Bioinformatics</b>, 2018, 19(5): 803-810. </p></li>
	<li class="testimonials__item"><p><b>Gao Y^</b>, Wang J^, Zheng Y^, Zhang J, Chen S & Zhao F. Comprehensive identification of internal structure and alternative splicing events in circular RNAs [J]. <b>Nature Communications</b>, 2016, DOI: 10.1038/ncomms12060. </p></li>
	<li class="testimonials__item"><p>Wang J^, <b>Gao Y^</b>, Zhao F. Phageâ€“bacteria interaction network in human oral microbiome [J]. <b>Environmental Microbiology</b>, 2016, 7(18): 2143-2158.</p></li>
	<li class="testimonials__item"><p><b>Gao Y^</b>, Wang J^, Zhao F. CIRI: an efficient and unbiased algorithm for de novo circular RNA identification [J]. <b>Genome Biology</b>, 2015, 16(1): 4. </p></li>
	<li class="testimonials__item"><p>Zhou H, Zhao H, Zheng J, <b>Gao Y</b>, Zhang Y, Zhao F & Wang J. CRISPRs provide broad and robust protection to oral microbial flora of gingival health against bacteriophage challenge. <b>Protein & Cell</b>, 2015, 6(7):541-545. </p></li>
	<li class="testimonials__item"><p>Ye N, Zhang X, Miao M, Fan X, Zheng Y, Xu D, Wang J, Zhou L, Wang D, <b>Gao Y</b>, Wang Y, Shi W, Ji P, Li D, Guan Z, Shao C, Zhuang Z, Gao Z, Qi J & Zhao F. Saccharina genomes provide novel insight into kelp biology [J]. <b>Nature Communications</b>, 2015, 6: 6986.  </p></li>
</ul></div></div></blockquote></section>


<!-- Process section--><div class="container"><section class="common-section" id="process">
<div class="advantages">
<h2>Opening Positions</h2><blockquote>
<p><b>Students</b></p>
<p>We have openings for motivated undergraduate/graduate students from various scientific disciplines (life sciences, mathematics/statistics, computer science, engineering, etc.) who wish to study on bioinformatics and computational genomics. Please contact gaoy@cas.ac.cn for more information.</p>
<p><b>Postdocs/Research associates</b></p>
<p>We have openings for experienced researchers in bioinformatics and computational genomics. Please contact <i><b>gaoy@cas.ac.cn</b></i> for more information.</p>
</blockquote></div></div></section>



<!-- Contacts section--><div class="contacts-section" id="contacts"><div class="container"><section class="contact-card">
<header class="contact-card__header"><h2>Contact Information</h2>
</header>
<div class="contact-info"><div class="contact-info__logo">PI</div>
<div class="contact-info__group">
	<div class="contact-info__additional">
		<p>China National Center for Bioinformation</p>
		<p>Beijing Institute of Genomics,</p>
		<p>Chinese Academy of Sciences</p>
	</div><div class="contact-info__vcard vcard"><h4 class="fn org">Prof. Yuan Gao</h4>
		<p class="role title">No.1 Beichen West Road</p><p class="icon icon_location">
		<a class="adr js-anchor" href="#gmap">
		<span class="country-name">Chaoyang District</span>,&nbsp;<span class="locality">Beijing</span>,&nbsp;<span class="address">China</span>
	   </a></p><p class="icon icon_mail"><a class="email" href="mailto:contact@example.com">gaoy@big.ac.cn</a></p></div></ul></div></div></section></div><div class="gmap" id="gmap"></div></div>


<!-- Footer--><footer class="main-footer"><div class="container"><p class="main-footer__copyright">&nbsp;<span></span></p><a class="main-footer__gotop icon icon_gotop js-anchor" href="#home">To top!</a><p class="main-footer__links"></p></div></footer><script src="static/js/body.min.js"></script></body></html>
